tests/models/bloom/test_modeling_bloom.py

```diff
@@ -416,6 +442,19 @@ def test_batch_generation(self):
     @slow
     @require_torch_gpu
     def test_batch_generation_padd(self):
+        # With small models the test will fail because of the operator torch.baddm that will give inconsistent results
```

Comment:
To be honest these statements are a bit confusing to me. I'd prefer to have stable tests in fp32 instead